modules/gitops-customizing-permissions-for-cluster-scoped-instances.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-customizing-permissions-for-cluster-scoped-instances.adoc{context}"]
+= Customizing permissions for cluster-scoped instances
+
+As a cluster administrator, to customize permissions for cluster-scoped instances, you must create new cluster roles and cluster role bindings for the {gitops-shortname} control plane components. 
+
+For example purposes, the following instructions focuses only on user-defined cluster-scoped instances.
+
+.Procedure
+
+. Open the *Administrator* perspective of the web console and go to *User Management* -> *Roles* -> *Create Role*. Use the following `ClusterRole` YAML template to add rules to specify the additional permissions.
++
+.Example cluster role YAML template
+
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: example-spring-petclinic-argocd-application-controller <1>
+rules:
+  - verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - '*'
+    resources:
+      - '*'
+  - verbs:
+      - '*'
+    apiGroups:
+      - ''
+    resources: <2>
+      - namespaces 
+      - persistentvolumes 
+----
+<1> The name of the cluster role according to the <argocd_name>-<argocd_namespace>-<control_plane_component> naming convention.
+<2> The resources to which you want to grant permissions at the cluster level.
+
+. Click *Create* to add the cluster role.
+
+. Find the service account used by the control plane component you are customizing permissions for, by performing the following steps:
++
+.. Go to *Workloads* -> *Pods*.
+.. Select the project where the user-defined cluster-scoped instance is installed from the *Project* drop-down menu.
+.. Click on the pod of the control plane component and go to *YAML* tab.
+.. Look for the `spec.ServiceAccount` field.
```

Comment:
Can you be more specific here? What should the user look for? Maybe try something like this: "Find the `spec.ServiceAccount` field and note the service account.: